javascript-light-bulb/main.js

```diff
@@ -1,10 +1,8 @@
 var $lightbulb = document.querySelector('.bulb-on');
 var $darklight = document.querySelector('.light-on');
-var counter = 0;
 
-function turnLightBulb() {
-  counter++;
-  if (counter % 2 === 0) {
+function turnLightBulb(value) {
+  if ($lightbulb.matches('div.bulb-off')) {
```

Comment:
This is now making your code dependent on the classes currently applied to DOM elements, which means your JavaScript is reliant on HTML to know what to do. You should follow my original suggestion of using a variable containing a boolean value to determine if the light is on or not.